rules-reviewed/camel3/camel2/xml-generic-information.windup.xml

```diff
@@ -30,7 +30,103 @@
 
                     </message>
                     <link href="https://camel.apache.org/manual/latest/camel-3-migration-guide.html#_zip_and_gzip_dataformat"
-                            title="Camel 3 - Migration Guide"/>
+                          title="Camel 3 - Migration Guide"/>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="xml-generic-information-00002">
+            <when>
+                <or>
+                    <xmlfile matches="//*/b:route/b:loadBalance/b:custom">
+                        <namespace prefix="b" uri="http://camel.apache.org/schema/blueprint"/>
+                    </xmlfile>
+                    <xmlfile matches="//*/c:route/c:loadBalance/c:custom">
+                        <namespace prefix="c" uri="http://camel.apache.org/schema/spring"/>
+                    </xmlfile>
+                </or>
+            </when>
+            <perform>
+                <hint title="`custom` load balancer was renamed `customLoadBalancer`" effort="1"
+                      category-id="mandatory">
+                    <message>`custom` load balancer was renamed `customLoadBalancer`
+
+                    </message>
+                    <link href="https://camel.apache.org/manual/latest/camel-3-migration-guide.html#_xml_dsl_migration"
+                          title="Camel 3 - Migration Guide"/>
+                    <quickfix type="REPLACE" name="camel.customLoadBalancerReplacementSpring">
+                        <replacement>c:customLoadBalancer</replacement>
+                        <search>c:custom</search>
+                    </quickfix>
+                    <quickfix type="REPLACE" name="camel.customLoadBalancerReplacementBluePrint">
+                        <replacement>b:customLoadBalancer</replacement>
+                        <search>b:custom</search>
+                    </quickfix>
```

Comment:
As `quickfix` does text searches I would suggest to replace the 2 quickfixes by 1 with this code : 
```
                    <quickfix type="REPLACE" name="camel.customLoadBalancerReplacement">
                        <replacement>customLoadBalancer</replacement>
                        <search>custom</search>
                    </quickfix>
```